app/views/dids/show.html.erb

```diff
@@ -22,6 +22,9 @@
         <%= attribute_row :terminated %>
         <%= attribute_row :billing_cycles_count %>
         <%= attribute_row :requirement, value: resource.requirement_link %>
+        <%= attribute_row :address_verification, value: @address_verification.link %>
```

Comment:
resource.address_verification_link

add this method to `DidDecorator` like others